ambry-clustermap/src/main/java/com.github.ambry.clustermap/StaticClusterManager.java

```diff
@@ -498,6 +502,29 @@ public void onReplicaEvent(ReplicaId replicaId, ReplicaEventType event) {
     }
   }
 
+  @Override
+  public JSONObject getSnapshot() {
+    JSONObject snapshot = new JSONObject();
+    snapshot.put(IMPLEMENTATION, StaticClusterManager.class.getName());
+    snapshot.put(CLUSTER_NAME, partitionLayout.getClusterName());
+    snapshot.put(TIMESTAMP_MS, SystemTime.getInstance().milliseconds());
+    JSONArray datacentersJsonArray = new JSONArray();
+    hardwareLayout.getDatacenters().forEach(dc -> {
+      JSONObject data = new JSONObject();
+      data.put(DATACENTER_NAME, dc.getName());
+      data.put(DATACENTER_ID, dc.getId());
+      JSONArray datanodesInDc = new JSONArray();
+      dc.getDataNodes().forEach(mockDataNodeId -> datanodesInDc.put(mockDataNodeId.getSnapshot()));
```

Comment:
Minor comment: should this be called mockDataNodeId given it's not a test class?  (Maybe was copied code.)

Reply:
Fixed